pcf/test/particle/aws/test_particles.py

```diff
@@ -23,7 +23,8 @@
 from pcf.particle.aws.emr.emr_cluster import EMRCluster
 from pcf.particle.aws.kms.kms_key import KMSKey
 from pcf.particle.aws.s3.s3_bucket import S3Bucket
-from pcf.particle.aws.vpc.vpc_instance import VPC
+from pcf.particle.aws.sns.sns_topic import SNSTopic
```

Comment:
dont need to import any particles anymore. have a pr already for removing them so no need to add  SNS.